[API][DOCS] custom path (#12412)
Sylius/Sylius: docs/api/unified_api/custom_paths.rst

Custom Paths in new Api
=======================

> All path in new API have the same prefix structure: ``/api/v2/admin/`` or ``/api/v2/shop/``

Reviewer: ```suggestion
All paths in new API have the same prefix structure: ``/api/v2/admin/`` or ``/api/v2/shop/``
```

> When you adding a new path to api resource configuration, you should remember to add also proper prefix.

Reviewer 2: ```suggestion
When you are adding a new path to api resource configuration, you should remember to add also proper prefix.
```

> When you are adding a new path to api resource configuration, you should remember to add also proper prefix.

Reviewer: ```suggestion
When you are adding a new path to API resource configuration, you should remember to add also proper prefix.
```

> You can declare entire path for each operation (without ``/api/v2/`` this part is configured globally):

Reviewer: ```suggestion
You can declare the entire path for each operation (without ``/api/v2/`` as this part is configured globally):
```

.. code-block:: xml

    <collectionOperation name="admin_get">
        <attribute name="method">GET</attribute>
        <attribute name="path">admin/orders</attribute>
    </collectionOperation>

> or you can add proper prefix for all path in chosen resource:

Reviewer: ```suggestion
or you can add a proper prefix for all paths in the chosen resource:
```

.. code-block:: xml

    <attribute name="route_prefix">shop</attribute>

.. note::

>     In some situations you may need add path with custom structure, in this cases you should configure it in security file (``ApiBundle/test/config/security.yaml``)

Reviewer 2: ```suggestion
    In some situations you may need to add a path with custom structure, in this case you should configure it in security file (``ApiBundle/test/config/security.yaml``)
```
or 'those cases' in plural form. But looking at context I would prefer this case ( just like in suggestion)

>     In some situations you may need to add a path with custom structure, in this case you should configure it in security file (``ApiBundle/test/config/security.yaml``)

Reviewer: ```suggestion
    In some situations, you may need to add a path with a custom structure, in this case, you will probably need to configure also the appropriate access in the security file (``ApiBundle/test/config/security.yaml``)
```
